test/unit/lib/much-rails/call-method_tests.rb

```diff
@@ -0,0 +1,61 @@
+require "assert"
+
+require "much-rails/call-method"
+
+module MuchRails::CallMethodTest
+  class UnitTests < Assert::Context
+    desc "MuchRails::CallMethod"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::CallMethod }
+
+    should "include MuchPlugin" do
+      assert_that(subject).includes(MuchPlugin)
+    end
+  end
+
+  class ReceiverTests < UnitTests
+    desc "receiver"
+    subject { receiver_class }
+
+    setup do
+      MuchStub.tap(subject, :new) { |new_record|
+        @new_class_method_called = true
+
+        MuchStub.(new_record, :call) {
+          @call_instance_method_called = true
+        }
+      }
+    end
+
+    let(:receiver_class) {
+      Class.new do
+        include MuchRails::CallMethod
+      end
+    }
```

Comment:
@jcredding are you cool using this `ReceiverTests` pattern when testing mix-ins?

Reply:
I'm cool with this and will look at switching anything I've already done to use it as well.